ConnectionManager.lib.nut

```diff
@@ -79,7 +82,7 @@ class ConnectionManager {
 	            break;
         	case START_DISCONNECTED:
```

Comment:
There is something wrong with indents in this switch block. Could you please fix it?